SIGs/GRC/publications/Confidential Payload Governance.md

```diff
@@ -0,0 +1,111 @@
+  * **Use secure credentials:** design SHOULD **\[b\]** avoid reliance on bearer tokens in authentication protocols.  
+      
+* **Secure & Attestable Build Environment**  
+    
+  * **Establish, Maintain and Produce Evidence of Integrity and Authenticity of the Build Tools and Environment:**  
+      
+    * **Secure and Isolated Build Practices:** The build process SHOULD **\[c\]** incorporate integrity and authenticity checks and strongly isolated compilation processes to prevent malicious tampering. The [SLSA **\[3\]** Build track](https://slsa.dev/spec/v1.0/levels#build-track) describes specific practices to enhance build process integrity. The build environment SHOULD **\[d\]** be integrity checked to detect tampering in the underlying compute system prior to running a build pipeline; the future [SLSA Build Environment track](https://slsa.dev/spec/draft/future-directions#build-environment-track) will define specific practices to achieve this, including the use of Confidential Computing (CC). CC or alternative data-in-use protection technology SHOULD be used to safeguard a build environment at runtime.  
+    * **Secure and Documented Build Tool Settings:** The build process MUST use recommended build tool settings as well as up-to-date versions of tools and dependencies. The configuration, inputs and outputs of the build process SHOULD be documented (e.g., via SLSA Provenance) and authenticated using a framework such as in-toto **\[4\]**. Failure to do either can lead to the generation of vulnerable executable code and make it more difficult to determine the integrity and trustworthiness of the produced code.  
+        
+  * **Reproducible Builds:** Builds SHOULD **\[f\]** be reproducible to ensure that the built binaries can be independently verified to match the source code. This approach provides additional measures for detecting tampering or unintended modifications during the build process.
+
+* **Secure and Attested Supply Chain Management**  
+    
+  * Automated tools MUST be used to continuously monitor and manage third-party software libraries and dependencies for known vulnerabilities. Dependencies MUST be kept updated and strict vetting processes applied, including code signature and build metadata checks if available, when ingesting dependencies. Dependency management MUST continue for the duration of the application’s deployment.  
+  * Include Bills of Materials as part of generated software; the BOMs SHOULD **\[g\]** be verified by the Verifier post-deployment.
```

Comment:
There is no 'g' (nor 7) in the Should v Must clarifications.

Reply:
Great catch. This needs to be added.

Reply:
@TheBankster Could you suggest the text for it?

Reply:
I will ask in Wednesday's GRC meeting; I am not sure actually. I don't believe I added those words.

Reply:
I am sorry we did not get to this in the GRC meeting. Can we leave this as an open issue?

Reply:
tracking this in https://github.com/confidential-computing/governance/issues/295